src/routes/routes.js

```diff
@@ -0,0 +1,97 @@
+import { Router } from 'express';
+import Joi from 'joi';
+import validator from 'express-joi-validation';
+
+import UsersController from '../controllers/controller';
+
+const controller = new UsersController();
+const router = Router();
+const userValidator = validator.createValidator({});
+
+const querySchema = {
+  userInfo: Joi.object({
+    login: Joi.string().required(),
+    password: Joi.string().required(),
+    age: Joi.number()
+      .min(4)
+      .max(130)
+      .required(),
+  }),
+  findUser: Joi.object({
+    id: Joi.string().required(),
+  }),
+};
+
+router
+  .get(
+    '/users/:id',
+    userValidator.params(querySchema.findUser),
+    (req, res) => {
+      const user = controller.getUser(req.params.id);
+
+      if (user) {
+        res.send(user);
+        return;
+      }
+
+      res.status(400).send('A user with this ID was not found.');
+    },
+  )
+  .put(
+    '/users/:id',
+    userValidator.query(querySchema.userInfo),
+    userValidator.params(querySchema.findUser),
+    (req, res) => {
+      const { id } = req.params;
+      const { login, password, age } = req.query;
+      //   const updatedUser = ;
+      if (controller.updateUser(id, login, password, age)) {
+        res.send(`${login} updated successfully.`);
+        return;
+      }
+
+      res.status(404).send('User was not found to update.');
+    },
+  )
+  .delete('/users/:id', (req, res) => {
+    if (controller.deleteUser(req.params.id)) {
+      res.send('User deleted successfully.');
+      return;
+    }
+    res.status(404).send('User not found to delete.');
+  })
+  .get('/users', (req, res) => {
+    const users = controller.showAllUsers();
+    if (users.length > 0) {
+      res.send(users);
+      return;
+    }
+
+    res.status(400).send('No users found.');
+  })
+  .post(
+    '/users',
+    userValidator.query(querySchema.userInfo),
+    (req, res) => {
+      const { login, password, age } = req.query;
```

Comment:
These fields should not be in `query` - they should be in `body`. Also, you should use `body-parser`

Reply:
updated